src/index.ejs

```diff
@@ -27,7 +27,7 @@
 </head>
 
 <body>
-  <div id="httpPath" style="display: none" th:text="<%='$'%>{domain}">${process.env.NODE_ENV === 'production' ? '' : 'http://localhost:9095'}</div>
+  <div id="httpPath" style="display: none" th:text="<%='$'%>{domain}">${process.env.NODE_ENV === 'production' ? '' : '.'}</div>
```

Comment:
Is there any special purpose here?

Reply:
Because there are already dev-http-path here: https://github.com/apache/shenyu-dashboard/blob/37763d94d06077f1170768980978da531546f328/.webpackrc.js#L50  
For the convenience of unified maintenance, the same configuration for the same target should be reduced.